src/SFA.DAS.ApprenticeFeedback.Web.AcceptanceTests/SFA.DAS.ApprenticeFeedback.Web.AcceptanceTests.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="NUnit" Version="3.13.1" />
     <PackageReference Include="NUnit3TestAdapter" Version="3.17.0" />
     <PackageReference Include="coverlet.collector" Version="3.0.2" />
+    <PackageReference Include="SFA.DAS.NServiceBus" Version="17.0.49" />
```

Comment:
I remember having an issue with trying to use the latest release version of this package a few months ago. I can't remember exactly what it was, probably some kind of version mismatch. Thought I should just mention.

Reply:
Ok - it is working I think though